backstopper-jaxrs/src/main/java/com/nike/backstopper/handler/jaxrs/JaxRsUnhandledExceptionHandler.java

```diff
@@ -9,26 +9,22 @@
 import com.nike.backstopper.model.DefaultErrorContractDTO;
 import com.nike.backstopper.model.util.JsonUtilWithDefaultErrorContractDTOSupport;
 
-import java.util.Collection;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-
 import javax.inject.Inject;
 import javax.inject.Singleton;
 import javax.ws.rs.core.Response;
+import java.util.*;
```

Comment:
Yeah, it's definitely a bikeshedding thing and everyone has their own preferences. Thanks for being flexible and conforming to our bikeshed. :)